frontend/src/components/FileUpload.js

```diff
@@ -1,29 +1,47 @@
 import {fileUpload} from "../services/fileUploadService";
-import {useState} from "react";
 import {useHistory} from "react-router-dom";
+import {IconButton, makeStyles} from "@material-ui/core";
+import AddCircleOutlineOutlinedIcon from "@material-ui/icons/AddCircleOutlineOutlined";
+
+const useStyles = makeStyles((theme) => ({
+    btnChoose: {
+        color: "var(--backgroundColorForms)",
+        bottom: theme.spacing(2),
+        right: theme.spacing(2),
+        position: "fixed"
+    }
+}))
 
 export default function FileUpload({onDraftReceived}) {
 
-    const [file, setFile] = useState(undefined);
-    const history =  useHistory();
+    const history = useHistory();
+    const classes = useStyles();
 
-    const onFormSubmit = (formEvent) => {
-        formEvent.preventDefault();
-        fileUpload(file).then((response) => {
+    const onChange = (changeEvent) => {
+        fileUpload(changeEvent.target.files[0]).then((response) => {
```

Comment:
Maybe rename to `uploadFile`, verb oriented naming style is preferable for functions. Please add error handling to prevent app from crashing on a network error.